tests/schemas/test_property_schema.py

```diff
@@ -41,6 +41,31 @@ def test_uniqueness_of_property_name(self, create_property):
 
         assert "name" in validation_errors
 
+    def test_property_can_update_with_same_name(self, create_property):
+        new_property = create_property()
+        payload = new_property.json()
+
+        payload["num_units"] = 42
+
+        context = {"name": new_property.name}
+
+        assert PropertyModel.update(
```

Comment:
I don't like hitting the database in these unit tests. We can accomplish the same tests with the validate method. There should be no need to hit the database in validation tests.

Other than that these tests look good. I think I would like one more for the sad path using the context. Since I don't think that line of code is being executed yet.

Reply:
Thanks for recommending a sad path test w/the context.  It actually exposed a bug in how I was assigning the context.  Updating now.

Reply:
@NickSchimek did you need me to also update these PostLoadDeserialization tests to use `validate()`, or do those need to hit the database?
https://github.com/codeforpdx/dwellinglybackend/blob/b9457c3c0b715cfe5dca4d6131a3fcfef1ed65e4/tests/schemas/test_property_schema.py#L70-L108

Reply:
Sure, They should be using `PropertySchema().load`.  Did I write these? I can't remember. I don't think anything in the schema test files should hit a method that is not part of the schema interface. `(validate, load, dump)`. There is actually very good reasoning behind this.

Reply:
Yes, you did write these 😆
I've updated all the tests in this file so that they use either `PropertySchema().load()` or `PropertySchema().validate()` now, so hopefully they are good to go 🤞